molecule/default/molecule.yml

```diff
@@ -0,0 +1,65 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: molecule/yamllint.yml
+platforms:
+  - name: north
+    groups:
+      - mta
+      - mda
+      - db
+      - test
+    image: minimum2scp/systemd-stretch
+    privileged: true
+    command: /sbin/init
+    networks:
+      - name: external
+
+  - name: south
+    groups:
+      - mta
+      - mda
+      - test
+    image: minimum2scp/systemd-stretch
```

Comment:
I don't know these images. Is there a specific reason to choose them over official images ?

Reply:
Good question, @HadrienPatte what do you think ? Why did you prefer those ?

Reply:
We need systemd-enabled images, and as far as I know, the official debian images are not.